README.md

```diff
@@ -1,14 +1,54 @@
+AmpliPi™ is a multi room/zone home audio controller and amplifier made for whole house audio systems with many zones. It can play up to 4 simultaneous audio sources, each of which can be selected from either its analog RCA input or any supported digital stream (Pandora, Spotify, AirPlay, etc). Each source can be routed to one or more zones, all of which are configurable in real-time using the self-hosted AmpliPi Web App or its underlying REST API. It is expandable up to 36 zones using AmpliPi Zone Expanders.
 
 The AmpliPi design is entirely open source, from the software, to the firmware, to the schematics. The REST API and Web App are written in Python and run on a Raspberry Pi 3+ Compute Module. The firmware, which provides the low level volume control and zone configuration, is written in C and configured by the Python API over I2C. All of the internal circuitry comes with full schematics (avaiable in this repo).
 
 The system is self-hosted on the Raspberry Pi and is privacy-centric. By design, AmpiPi doesn’t listen to you or spy on you — it just plays your audio! The way it should be. An internet connection is only needed for some external streaming sources, such as Pandora. The Python application running on the Pi hosts a mobile-friendly website and REST API used to control the system. The website is built on top of the REST API.
 
+![High-level Schematic](docs/imgs/Signal_Diagram_with_Expansion_Out.png)
+
 Check us out on [Kickstarter](https://www.kickstarter.com/projects/micro-nova/amplipi-home-audio-system)!
 
+## Features
+
+AmpliPi can play many different types of digital streaming inputs. Most of the streaming services supported can be played as independent digital streams on all four sources at once, check out the **Multiple?** heading. Below is the current status of our digital stream integrations.
+
+|Streaming Service|Supported|Multiple?|Metadata|Notes|
+|--|--:|--:|--:|--|
+|Pandora|Yes|Yes|Yes||
+|Airplay|Yes|Yes|Yes|Metadata only available from itunes|
+|Spotify|Yes|Yes|No see [#37](http://github.com/micro-nova/AmpliPi/issues/37)|Requires Spotify Premium, one account per stream|
+|DLNA|Yes|Yes|Yes||
+|Internet Radio|Yes|Yes|Yes||
+|PlexAmp|Yes|No|No||
+|Google Cast|No|||Need to investigate|
+|Offline Music|No|||See [#66](http://github.com/micro-nova/AmpliPi/issues/66)|
+|TIDAL|No|||See [#87](http://github.com/micro-nova/AmpliPi/issues/87)|
+
+In the future we plan to integrate with several home automation systems. Below is their current status.
+
+|Integrations|Supported|Notes|
+|--|--:|--|
+|Home Assistant|No|Future integration planned|
+|Open HAB|No|See [#54](http://github.com/micro-nova/AmpliPi/issues/54)|
+|Alexa|No|See [#25](http://github.com/micro-nova/AmpliPi/issues/25)|
+|Google Assistant|No|See [#69](http://github.com/micro-nova/AmpliPi/issues/69)|
+
+## Quick Start Guide
+
+If you are one of the lucky few to have a pre-release version of the AmpliPi.
```

Comment:
lines 41 and 42 shouldn't be separate sentences IMO; I would put a comma between AmpliPi and see [doc] to get started.